tests/test_integration.py

```diff
@@ -1,9 +1,9 @@
 from dotenv import load_dotenv, find_dotenv
 from unittest.mock import patch, Mock
-from datetime import datetime
+from datetime import date
+from item import Item
 import pytest
 import app
-import os
 
 @pytest.fixture
 def client():
```

Comment:
It would be worthwhile explicitly setting the LOGIN_DISABLED environment variable before creating the app here so that that's set appropriately for the tests